src/PHPMailer.php

```diff
@@ -3857,6 +3857,10 @@ protected function serverHostname()
      */
     public static function isValidHost($host)
     {
+        if (version_compare(PHP_VERSION, '7.0.0') >= 0) {
```

Comment:
While I know that this constant was added in 7.0.0, I prefer to check for the feature directly, so `defined('FILTER_VALIDATE_DOMAIN')` instead of `version_compare` check. It's more specific, and a little safer (see also why we needed to do this for checking whether the openssl extension was loaded) .

Reply:
Clear, fixed this.